libimgui-docking/README.md

```diff
@@ -0,0 +1 @@
+../upstream/docs/README.md
```

Comment:
Minor:
Currently this symlink references a file from `upstream`, however I think it should reference `upstream-docking`